kubejs/server_scripts/base/mods/minecraft.js

```diff
@@ -476,3 +476,54 @@ if (feature("Wax recipes")) {
   );
   addFilling("minecraft:honeycomb", "ae2:silicon", "750x create:honey");
 }
+
+if (feature("Book back into paper")) {
+  addCutting("3x paper", "book");
+}
+
+if (feature("Spider eye with manapool")) {
+  addInfusion("spider_eye", "red_dye", 1250, "minecraft:grass_block");
+}
+
+if (feature("Blaze rod production")) {
+  addInfusion("blaze_rod", "stick", 20000, "minecraft:magma_block");
+}
+
+if (feature("Remove none oak boats")) {
```

Comment:
If they're the only boat in the game. Why not rename it to just boat than `Applewood boat`  in a lang file

Reply:
True, will do that